### DISCUSSION TITLE
Need to rerun all steps every time? all datas and folder are overwritten?

### DISCUSSION THREAD

**Discussion by Mr-Jay** [TheLastBen/fast-stable-diffusion#141] [Q&A]
Hi, i'm a new user of this colab,
from the few try i did today it's look like every time the runtime disconnect, we need to rerun everything? 

My hypernetwork is gone also so it's look like every datas aren't kept between runtime. 
Even all models installed manually in the proper folder are deleted (actually still visible in my gdrive, but not visible in a new colab runtime).

Before to use this version i made my own, where i only needed to run the main code to launch automatic, but was quite instable (crash if  generate more than 1 image), is there a way to do same? 

Have one installation set up to do only once
a pull set up to do once a while

And only the main code to launch automatic with only 1 click every time? 

All datas like models, training, hypernetwork, output... are savec  between runtime. 

Cheers

**Comment by TheLastBen** [comment]
Nothing is getting deleted by this colab, if there is an existing installation of the webui, it will not touch it.
The only case it pulls the repo is when you check the box "update repo", and even then it will not delete anything.

**Reply by Mr-Jay** [reply]
That's weird, because none of the previous runtime has been kept, i will try again tomorrow to see if i'm losing today's set up .

Just for confirmation, i need to run all the "installing" cells and model download, every time before to launch automatic?

**Reply by TheLastBen** [reply]
Yes, all the cells. After the first run, anytime you run the colab will take less than 30 seconds to get to the last cell
